### ISSUE TITLE
Try without except/finally has no error

### ISSUE THREAD

**Issue by jakebailey** [microsoft/pyright#1513]
Code like:

```python
def func():
    try:
        pass
```

```python
def func():
    try:
        pass

	return 1234
```

Doesn't emit any syntax error diagnostics in pyright/pylance, but Python gives a syntax error.

**Comment by erictraut** [comment]
Good catch.

**Comment by erictraut** [comment]
This is addressed in Pyright 1.1.114, which I just published. It will also be in the next release of Pylance.
